charts/oam-core-resources/templates/oam-local-controller.yaml

```diff
@@ -96,7 +16,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: {{ include "oam-core-resources.fullname" . }}
+  name: "cluster-admin"
```

Comment:
why change here?

Reply:
The trait controller now has to get/list any resources a workload has, we can't use a pre-defined role with a finite resource list. There is a similar problem with the appConfig as one needs to add a cluster role and role binding to every new resource we introduce (like KNative).

Reply:
I see. Thanks

Reply:
So the cluster-admin is an internal role, right?

Reply:
I am afraid user might use the same `ClusterRole` name.

Reply:
> So the cluster-admin is an internal role, right?

yes

Reply:
> I am afraid user might use the same `ClusterRole` name.

I wonder who are the "user" referring to here?